src/server/StudioManagementSystem/Mappers/OwnerContactMapper.cs

```diff
@@ -5,8 +5,10 @@ namespace StudioManagementSystem.Mappers;
 
 public static class OwnerContactMapper
 {
-    public static OwnerDto MapToOwnerDto(this OwnerContact owner)
+    public static OwnerDto? MapToOwnerDto(this OwnerContact? owner)
     {
+        if (owner == null)
+            return null;
```

Comment:
Adding logic like this to the mapper only propagates an issue that should be handled further downstream. What originally returned a null value to the mapper?